ambry-router/src/main/java/com.github.ambry.router/GetBlobOperation.java

```diff
@@ -305,6 +310,7 @@ public long getSize() {
       if (operationException.get() != null) {
         completeRead();
       }
+      readyForPollCallback.onPollReady();
```

Comment:
But then why are we ready for a poll callback? you don't need to get any new chunks from the server yet.